tests.py

```diff
@@ -404,11 +530,102 @@ def test_get_projects_include_deleted_revisions(self):
         # Send it
         ts.get_projects(revisions=True, include_deleted=True)
 
+        patched_json_loader.stop()
+
         # Test that requests.get was called with correct paramaters
         requests.get.assert_called_with("http://ts.example.com/v1/projects"
                                         + "?include_deleted=true"
                                         + "&revisions=true")
 
+    def test_json_to_python_single_object(self):
+        """Test that TimeSync._json_to_python converts a json object to a python
+        list of object"""
+        baseurl = 'http://ts.example.com'
+        # Instantiate timesync class
+        ts = pymesync.TimeSync(baseurl,
+                               password="password",
+                               user="example-user",
+                               auth_type="password")
+
+        json_object = '{\
+            "uri": "https://code.osuosl.org/projects/ganeti-webmgr",\
+            "name": "Ganeti Web Manager",\
+            "slugs": ["ganeti", "gwm"],\
+            "owner": "example-user",\
+            "uuid": "a034806c-00db-4fe1-8de8-514575f31bfb",\
+            "revision": 4,\
+            "created_at": "2014-07-17",\
+            "deleted_at": null,\
+            "updated_at": "2014-07-20"\
+        }'
+
+        python_object = [
+            {
+                u'uuid': u'a034806c-00db-4fe1-8de8-514575f31bfb',
+                u'updated_at': u'2014-07-20',
+                u'created_at': u'2014-07-17',
+                u'uri': u'https://code.osuosl.org/projects/ganeti-webmgr',
+                u'name': u'Ganeti Web Manager',
+                u'owner': u'example-user',
+                u'deleted_at': None,
+                u'slugs': [u'ganeti', u'gwm'],
+                u'revision': 4
+            }
+        ]
+
+        self.assertEquals(ts._json_to_python(json_object), python_object)
+
+    def test_json_to_python_list_of_object(self):
+        """Test that TimeSync._json_to_python converts a json list of objects
+        to a python list of objects"""
+        baseurl = 'http://ts.example.com'
+        # Instantiate timesync class
+        ts = pymesync.TimeSync(baseurl,
+                               password="password",
+                               user="example-user",
+                               auth_type="password")
+
+        json_object = '[\
```

Comment:
This looks like a single time object broken up into multiple objects - this will work fine to compare outputs, but it may be a little easier for developers to grok if it reflects the actual structure we are expecting from TimeSync. I realize thats a lot more verbose, but I think it might be worth it for clarity and consistency.